EIPS/eip-5792.md

```diff
@@ -122,38 +136,55 @@ Returns the status of a call batch that was sent via `wallet_sendCalls`. The ide
 type GetCallsParams = string;
 
 type GetCallsResult = {
-  status: 'PENDING' | 'CONFIRMED';
+  batchId: `0x${string}`;
```

Comment:
callsId

Reply:
should keep the name the same

Reply:
Done.

Reply:
Super nit: kinda think `id` is even sufficient enough and maybe more consistent (ie. we don’t call it `transactionHash` on the transaction schema)